llms/hf_llm/generate.py

```diff
@@ -43,6 +43,9 @@ def generate(
         s = tokenizer.decode(tokens)
         print(s[skip:], end="", flush=True)
         skip = len(s)
+    if len(tokens) == 0:
+        print("No tokens generated for this prompt")
+        return
```

Comment:
I love it. But could you put it below the line which prints the `"="*10` ? That way it's super clear that it's not the model's output!

Reply:
updated in https://github.com/ml-explore/mlx-examples/pull/231/commits/53ee382cf850cdb547e2fb345e0cff5e59836a05